### PR TITLE
Remove profiler

### PR DESCRIPTION
**Pull request by RyanCavanaugh**
##### Bug

Expected: Profiler is not present
Actual: Is present

##### Fix

Remove profiler

##### Testing

Try to launch profiler. Verify it is not present

cc @billti @paulvanbrenk @mjbvz

### CODE REVIEW

File: Nodejs/Product/AssemblyVersion.cs [hunk 1]
```diff
@@ -39,7 +39,7 @@ class AssemblyVersionInfo {
 
     // This version should never change from "4100.00"; BuildRelease.ps1
     // will replace it with a generated value.
-    public const string BuildNumber = "4100.00";
+    public const string BuildNumber = "41115.00";
```
Comment (RyanCavanaugh) [inline Nodejs/Product/AssemblyVersion.cs]: Should I revert this? It changed on its own

**Comment by paulvanbrenk** [comment]
LGTM

File: Nodejs/Product/AssemblyVersion.cs (hunk 1 above)
Comment (paulvanbrenk) [inline Nodejs/Product/AssemblyVersion.cs]: Yep, the comment says it should never change.

**Comment by paulvanbrenk** [comment]
It's gone

**Pull request closed without merging**